docs/codeql/codeql-language-guides/using-api-graphs-in-python.rst

```diff
@@ -31,9 +31,9 @@ following snippet demonstrates.
 
 ➤ `See this in the query console on LGTM.com <https://lgtm.com/query/1876172022264324639/>`__.
 
-This query only selects the API graph node corresponding to the ``re`` module. To find
-where this module is referenced, you can use the ``getAUse`` method. The following query selects
-all references to the ``re`` module in the current database.
+This query selects the API graph node corresponding to the ``re`` module. This node represent the fact that the ``re`` module has been imported rather than a specific place in the program where the import happens. Therefore, there will be at most one result per project, and it will not have a useful location, so you have to click `Show 1 non-source result` in order to see it.
+
+To find places in the program where the ``re`` module is referenced, you can use the ``getAUse`` method. The following query selects all references to the ``re`` module in the current database.
```

Comment:
Feel free to ignore - I just find odd the use of "places in the program", so suggesting an alternative.

```suggestion
To find where the ``re`` module is referenced in the program, you can use the ``getAUse`` method. The following query selects all references to the ``re`` module in the current database.
```